intelmq/bots/BOTS

```diff
@@ -728,6 +728,18 @@
                 "port": "<port>"
             }
         },
+        "Redis": {
+            "description": "Redis is the bot responsible to send events to a Redis server.",
+            "module": "intelmq.bots.outputs.redis.output",
+            "parameters": {
+                "redis_server_ip": "127.0.0.1",
+                "redis_server_port": "6379",
+                "redis_db": "2",
+                "redis_queue": "external-redis-queue",
+                "redis_timeout": "50000",
```

Comment:
But if I remove the "" it does not pass... also, other bots are using the strings... or I'm missing something?
E.g:
"Reverse DNS": {
            "description": "Reverse DNS is the bot resposible to get the correspondent domain name source IP and destination IP of the events",
            "module": "intelmq.bots.experts.reverse_dns.expert",
            "parameters": {
                "redis_cache_db": "5",
                "redis_cache_host": "127.0.0.1",
                "redis_cache_port": "6379",
                "redis_cache_ttl": "86400"
            }
        },